Assets/MRTK/Core/Inspectors/Setup/MixedRealityProjectConfiguratorWindow.cs

```diff
@@ -364,22 +368,24 @@ private void RenderSelectXRSDKPlugin()
             CreateSpace(15);
 
             if (GUILayout.Button("\n<b>Unity OpenXR plugin (recommended)</b><size=4>\n\n</size>"
-            + "Choose this if you want to embrace the new industry standard and easily support a wide range of AR/VR devices in the future! Currently officially supports HoloLens 2 and Windows Mixed Reality headsets with other devices coming soon.\n", multiLineButtonStyle))
+            + "Choose this if you want to embrace the new industry standard and easily support a wide range of AR/VR devices in the future! Currently officially supports HoloLens 2 and Windows Mixed Reality headsets with other devices coming soon. The Unity OpenXR Plugin will be installed.\n", multiLineButtonStyle))
             {
 
 #if UNITY_2020_2_OR_NEWER
+                Debug.Log(MRTKConfiguratorLogPrefix + " Installing the Unity OpenXR Plugin. Operation performed is equivalent to installing the OpenXR plugin manually via Window -> Package Manager -> Packages: Unity Registry.");
                 AddUPMPackage("com.unity.xr.openxr");
                 CurrentStage = ConfigurationStage.InstallOpenXR;
 #endif // UNITY_2020_2_OR_NEWER
                 Repaint();
             }
             CreateSpace(15);
             if (GUILayout.Button("\n<b>Built-in Unity plugins (non-OpenXR)</b><size=4>\n\n</size>"
-            + "Choose this if your application needs to support platforms beyond just HoloLens 2 and Windows Mixed Reality headsets (e.g. Oculus/Magic Leap headsets which are currently not supported via OpenXR).\n", multiLineButtonStyle))
+            + "Choose this if your application needs to support platforms beyond just HoloLens 2 and Windows Mixed Reality headsets (e.g. Oculus/Magic Leap headsets which are currently not fully supported via OpenXR). The Unity XR Management Plugin will be installed if not already.\n", multiLineButtonStyle))
```

Comment:
nit, but I think Oculus headsets are now more properly supported by Unity in the latest release:

![image](https://user-images.githubusercontent.com/3580640/119169605-125c8700-ba17-11eb-9f1e-5bd1a2e988ad.png)

We still need to add support for the Oculus Touch controller via OpenXR in MRTK though